sqe-http-api/DTOs/ImagedObject.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
```

Comment:
What is this for?

Reply:
My IDE tells me that of all those usings, the only one used by the code in that namespace is: using System.Collections.Generic;

I have gone through and cleaned out the unused "usings" (~50 occurences).